Managers/NetworkServer.h

```diff
@@ -259,7 +259,10 @@ namespace RTE {
 
 		unsigned char m_SceneID; //!<
 
-		bool m_ResetActivityVotes[c_MaxClients]; //!<
+		bool m_EndActivityVotes[c_MaxClients]; //!< Votes from each player required to return to the Multiplayer Lobby.
+		bool m_RestartActivityVotes[c_MaxClients]; //!< Votes from each player required to restart the current activity.
+
+		long long m_LatestRestartTime; //!< 
```

Comment:
If it's not difficult to figure out, please chuck in a member variable comment for LatestRestartTime. Don't worry too much though, as I've mentioned before, the current multiplayer implementation is gonna get deleted in future.